dangerzone/gui/__init__.py

```diff
@@ -85,50 +86,42 @@ def delete_window(window_id: str) -> None:
         del windows[window_id]
 
     # Open a document in a window
-    def select_document(filename: Optional[str] = None) -> bool:
-        if (
-            len(windows) == 1
-            and windows[list(windows.keys())[0]].document.input_filename == None
-        ):
-            window = windows[list(windows.keys())[0]]
```

Comment:
If I understand correctly, we now remove this logic whereby only one window could get created. This should be documented in the commit message that introduces this removal, since currently we are just saying we are simplifying the new window logic.

Reply:
I don't think so. I think here the goal was really just to simplify the logic. It can be quite convoluted to understand but from my understanding, before it was checking if there was already a window open without a selected document. If there was, it would not create a window. If there wasn't then it would create a new window.

This check didn't seem important as what is the consequence of the user starting multiple windows without.

Reply:
I see. I didn't realize there could be a case where you have a window open without a document. If it's not important then sure, it can go.